src/type_check.rs

```diff
@@ -179,6 +187,7 @@ impl IPDLType {
 
         match itype {
             IPDLType::ActorType(_, _) => (),
+            IPDLType::ImportedCxxType(_, Lifetime::RefCounted, _, _) => (),
```

Comment:
(Note: this came from bug 1607634, unlike the rest of the patch which came from bug 1812156)

Reply:
In my pull request, I split this change out into a separate diff, and implemented the rest of bug 1812156.